extensions/import/src/wizard/pages/modifyColumnsPage.ts

```diff
@@ -120,7 +122,14 @@ export class ModifyColumnsPage extends ImportPage {
 
 		return true;
 	}
-
+	private setupNavigationValidator() {
+		this.instance.registerNavigationValidator((info) => {
+			if (this.loading.loading) {
```

Comment:
nit: I think this code block (127-130) could be written `return !this.loading.loading;`

Reply:
The only reason I kept it like this was in case more cases were added in the future. However I'll change it to be less verbose.

Reply:
@aaomidi just a comment on style, totally up to you if you want to update the code 😄